lighthouse-cli/bin.js

```diff
@@ -58,16 +62,22 @@ async function begin() {
     commands.listTraceCategories();
   }
 
-  const url = cliFlags._[0];
+  const urlUnderTest = cliFlags._[0];
 
   /** @type {LH.Config.Json|undefined} */
   let configJson;
   if (cliFlags.configPath) {
     // Resolve the config file path relative to where cli was called.
     cliFlags.configPath = path.resolve(process.cwd(), cliFlags.configPath);
-    configJson = require(cliFlags.configPath);
+
+    if (cliFlags.configPath.endsWith('.json')) {
+      configJson = JSON.parse(fs.readFileSync(cliFlags.configPath, 'utf-8'));
+    } else {
+      const configModuleUrl = url.pathToFileURL(cliFlags.configPath).href;
+      configJson = (await import(configModuleUrl)).default;
+    }
   } else if (cliFlags.preset) {
-    configJson = require(`../lighthouse-core/config/${cliFlags.preset}-config.js`);
+    configJson = (await import(`../lighthouse-core/config/${cliFlags.preset}-config.js`)).default;
```

Comment:
this seems like a place where a `createRequire` would be justified. Pretty awkward without it

Reply:
wouldn't that fail for es modules?

Reply:
oh yeah, good point :(